docs/about-lightning.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Lightning (Coming soon)
+sidebar_label: Lightning
+---
+
+## The Lightning pitch
+
+##### Leveraging the tech powering the field-tested enterprise OpenFn platform...
+
+Lightning brings together the tried and tested technology we have been using
+since 2015 (OpenFn core, engine, adaptors and code from the OpenFn SaaS
+platform) to manage the orchestration and execution of integrations in a stable,
+scalable and secure way.
+
+##### ...but providing a fully open source web app with a user-friendly web interface.
+
+A fully open source web app, it can be deployed anywhere through Docker and
+comes with a user-friendly, low-code interface with the full functionality
+needed for organizations to build, run and audit their workflows all in one
+place. ![Lightning](/img/lightning_build_run_audit.png)
+
+##### Build
+
+Empower more users in your organization to have a say in what gets automated and
+how. Lightning’s visual interface makes workflows more intelligible to
+non-technical users, bridging the gap between the IT specialists that build out
+automations and program managers that are the real business/ program experts on
+the processes that need automating.
+
+##### Audit
+
+Treat every workflow run with the care and attention it deserves. In OpenFn,
+each incoming request or transaction that gets processed is more than a piece of
+data - it represents a vulnerable child in need of critical support, a farmer
+managing their savings to make sure they can afford the next harvest. Lightning
+provides users with a dashboard that allows them to monitor the health of their
+integrations to make sure no request goes unprocessed.
+
+## What is Lightning ?
```

Comment:
We've already said (in a different way) what Lightning is above. Let's explain "what is Lightning" one time, and then use subsequent sections to dig more into details... so maybe here in this section we want to talk more specifically about `Lightning as a Workflow Engine` or something like that. 
But this section does not flow with the above, seems redundant.